actions/java-gradle-test/action.yaml

```diff
@@ -54,7 +58,12 @@ runs:
         gradle-cache: ${{ inputs.gradle-cache }}
 
     - name: Run tests
-      run: ./gradlew check --info --stacktrace
+      run: |
+        if [[ ${{ inputs.integration-tests }} == true ]]; then
+          ./gradlew check --info -PincludeTests="**/*IntegrationTest*"
```

Comment:
You are right. I'm closing this MR since you can just use env variables as well, which keeps the action more flexible anyway, in this case, e.g.:
```
env: 
  ORG_GRADLE_PROJECT_includeTests: "**/*IntegrationTest*"
```